src/electionguard-ui/ElectionGuard.UI/Models/TallyCeremonyChecklist.cs

```diff
@@ -43,10 +45,11 @@ public partial class TallyCeremonyChecklist : ObservableObject
         int sharesComputed,
         int challengesResponded)
     {
-        State = tally.State;
         _quorum = tally.Quorum;
         _guardiansJoined = guardiansJoined;
         _sharesComputed = sharesComputed;
         _challengesResponded = challengesResponded;
+        QuorumReached = _guardiansJoined >= _quorum;
+        State = tally.State;

```

Comment:
Moved State to the end of the list since it signals that all of the other properties have been changed.